osgbuild/importer.py

```diff
@@ -449,6 +458,11 @@ def main(argv=None):
             help="If there is an osg/ directory, do a 3-way diff like --diff3-spec.  Otherwise just update"
             " the .source file in the 'upstream' directory."
         )
+        parser.add_option(
+            "--nosha1sum", action="store_false", dest="sha1sum", default=True,
```

Comment:
How about making the `dest` option name `want_sha1sum`, to clarify that it is a bool, and not the same as the other `sha1sum` arguments to functions, which are meant to be strings?

(tbh, because of this `dest="sha1sum", default=True`, i was confused in earlier functions that took a `sha1sum` argument whether they took a bool or a string.)

Reply:
Sounds good.